en/02_Developer_Guides/02_Controllers/03_Access_Control.md

```diff
@@ -14,6 +14,9 @@ actions on the website they shouldn't be able to.
 Any action you define on a controller must be defined in a `$allowed_actions` configuration array. This prevents users from
 directly calling methods that they shouldn't.
 
+> [!NOTE]
+> If the permission check fails, Silverstripe CMS will return a `403` Forbidden HTTP status.
```

Comment:
Moved this from below to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.